bin/compile

```diff
@@ -10,12 +10,12 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 VENDOR_DIR="$BUILD_DIR/vendor"
 INSTALL_DIR="$VENDOR_DIR/imagemagick"
-CACHE_FILE="$CACHE_DIR/imagemagick.tar.gz"
-USE_CACHE_FILE='yes'
+IMAGE_MAGICK_VERSION="${IMAGE_MAGICK_VERSION:-6.9.5-5}"
```

Comment:
:+1:

Out of curiousity, do you know what this type of string expansion is called?